### PR TITLE
[Resource]: Upload Fullstack Node.js Book in English

### PR DESCRIPTION
**Pull request by mohdahsanrazakhan**
## What does this PR do?
Adding new Node book in English book lists. 

### Description
This book contains everything you need to learn to create fast, production [Node.js](http://en.wikipedia.org/wiki/Node.js) code. Including (but not limited to): How to create high-performance API servers in Node, How to leverage Node's async capabilities for maximum performance, Community conventions for organize Node.js code, Storing data in a database and exposing it via an API.

### Why is this valuable (or not)?
Yes, it's a valuable resource for learning how to create high-performance API servers in Node.js, leverage its async capabilities, follow community conventions, and handle data storage and API exposure efficiently.

### How do we know it's really free?
This book is shared by [FreeComputerBooks.com](https://freecomputerbooks.com/Fullstack-Node.js.html)

### For book lists, is it a book?
Yes, it is a book for book lists.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1331,6 +1331,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Shell scripting with Node.js](https://exploringjs.com/nodejs-shell-scripting/index.html) - Axel Rauschmayer (HTML)
 * [The Node Beginner Book](http://nodebeginner.org)
 * [The Node.js Handbook](https://flaviocopes.com/page/node-handbook/) - Flavio Copes (PDF, EPUB, Kindle) *(email address requested)*
+* [Fullstack Node.js: The Complete Guide to Building Production Apps with Node.js](https://moscow-city.guide/upload/iblock/8ce/8ceee361e68ff14b4e2aa66575be659d.pdf) - David Gutman, Nate Murray (PDF)
```
Comment (rng70) [inline books/free-programming-books-langs.md:1334]: move this line to line `1319`

**Review by rng70 — COMMENTED** [review]
Moving this line to line 1319 will resolve the linter issue

File: books/free-programming-books-langs.md (hunk 1 above)
Comment (mohdahsanrazakhan) [reply books/free-programming-books-langs.md]: Yes @rng70 you are right, I changed it.

**Review by rng70 — APPROVED** [review]
LGTM to merge

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1316,6 +1316,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Essential Node.js](https://www.programming-books.io/essential/nodejs/) - Krzysztof Kowalczyk, StackOverflow Contributors
 * [From Containers to Kubernetes with Node.js](https://www.digitalocean.com/community/books/from-containers-to-kubernetes-with-node-js-ebook) - Kathleen Juell (PDF, EPUB)
 * [Full Stack JavaScript: Learn Backbone.js, Node.js and MongoDB](https://github.com/azat-co/fullstack-javascript) - Azat Mardan
+* [Full Stack Node.js: The Complete Guide to Building Production Apps with Node.js](https://moscow-city.guide/upload/iblock/8ce/8ceee361e68ff14b4e2aa66575be659d.pdf) - David Gutman, Nate Murray (PDF)
```
Comment (sohansai) [inline books/free-programming-books-langs.md:1319]: This is not an authorized distribution. Refer to docs/CONTRIBUTING.md for guidelines.

Comment (mohdahsanrazakhan) [reply books/free-programming-books-langs.md:1319]: I think you are talking about the point 4 which is given below:
* use the link with the most authoritative source (meaning the author's website is better than the editor's website, which is better than a third-party website)

This point showing us a priority, not denying to not upload the third-party website. Right?

**Pull request closed without merging**